lib/default-config.js

```diff
@@ -1050,7 +1050,7 @@ module.exports = function (config) {
       choices = choices.slice(0);
 
       // Array of choice arrays should be flattened.
-      choices = _.flatten(choices);
+      choices = _.compact(_.flatten(choices));
```

Comment:
Ahhh yikes, yeah I didn't run it in the demo, I `npm link`ed it. I'll be sure to use the demo moving forward.